doc-sites/antora-playbook.yml

```diff
@@ -7,10 +7,30 @@ site:
   start_page: james-site::homepage.adoc
 content:
   sources:
-    - url: https://github.com/apache/james-project.git
+    - url: https://github.com/chibenwa/james-project.git
```

Comment:
should be apache repo no? Idem for other items?

Reply:
Correct. I WIP this then.

Reply:
The versioning fix PRs have been merged on all concerned branches in Apache james repo, so I guess we can switch to apache repo here, and it should build us the proper antora doc we want correct? :)